### ISSUE TITLE
Enhancement: Create Event multi-select on calls.

### ISSUE THREAD

**Issue by Squirrelies** [openmhz/trunk-server#170]
When attempting to create an event with 50+ calls, it can be a pain to drag and drop each individual call onto the create event section. It would be nice if there was a selection checkbox or the ability to drag select, or the ability to shift-click a range of calls into the create event section. I recently had an event that spanned 22:54 to 00:40 and produced a great deal of calls.

Tl;dr multi-select for event creation would be nice. :)

**Comment by robotastic** [comment]
Agreed - I think this would make the workflow a lot easier. Let me see what I can do.

**Comment by robotastic** [comment]
I think this should be live... its not perfect, but let me know what you think
